File: kalkulator.cs
using System;

class Kalkulator {
    static void Main() {
        Console.Write("Podaj pierwszą liczbę: ");
        double a = double.Parse(Console.ReadLine());
        Console.Write("Podaj drugą liczbę: ");
        double b = double.Parse(Console.ReadLine());
        Console.Write("Wybierz operację (+, -, *, /): ");
        string op = Console.ReadLine();

        double wynik = 0;
        switch (op) {
            case "+": wynik = a + b; break;
            case "-": wynik = a - b; break;
            case "*": wynik = a * b; break;
            case "/":
                if (b != 0) wynik = a / b;
                else { Console.WriteLine("Błąd: dzielenie przez zero!"); return; }
                break;
            default:
                Console.WriteLine("Nieznana operacja!"); return;
        }
        Console.WriteLine($"Wynik: {wynik}");
    }
}

File: kalkulator.py
a = float(input("Podaj pierwszą liczbę: "))
b = float(input("Podaj drugą liczbę: "))
operacja = input("Wybierz operację (+, -, *, /): ")

if operacja == "+":
    wynik = a + b
elif operacja == "-":
    wynik = a - b
elif operacja == "*":
    wynik = a * b
elif operacja == "/":
    if b != 0:
        wynik = a / b
    else:
        print("Błąd: dzielenie przez zero!")
        wynik = None
else:
    print("Nieznana operacja!")
    wynik = None

if wynik is not None:
    print(f"Wynik: {wynik}")
